packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,525 +1,527 @@
+## 2.1.16
+
+- Add on complete callback
+
 ## 2.1.15
 
-* Ensured seekTo isn't called before video player is initialized. Fixes [#89259](https://github.com/flutter/flutter/issues/89259).
```

Comment:
Please do not rewrite the entire CHANGELOG format.

Reply:
I just revert the change